index.html

```diff
@@ -8,9 +8,80 @@
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.9/css/all.css" integrity="sha384-5SOiIsAziJl6AWe0HWRKTXlfcSHKmYV4RBF18PPJ173Kzn7jzMyFuTtk8JA7QQG1" crossorigin="anonymous">
+
 </head>
 <body>
 
+        <nav class="navigation">
+     <div class="content">
+       <ul class="links">
+         <li class="active">Meet Karma</li>
+         <li>How it Works</li>
+         <li>Store</li>
+         <li>Blog</li>
+         <li>Help</li>
+         <li>Login</li>
+       </ul>
+       <img class="logo" src="img/karma-logo.svg"  width="26">
+     </div>
```

Comment:
if the logo appears first, from left-to-right, then it should (in most cases) appear first in your HTML. 
Try moving it before the `<ul>` and you won't need the negative margin anymore: https://github.com/CodeYourFuture/karma-clone/pull/2/files#diff-6a6a912d0c7b55b537768da778032849R44

Reply:
Also, try adding `display: flex;` to the `.content` div and position the logo and nav links using flexbox 😉